client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx

```diff
@@ -106,6 +107,21 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow } ) => {
 	const categorizationOptions = getCategorizationOptions( intent, true );
 	const categorization = useCategorization( staticDesigns, categorizationOptions );
 
+	// TODO: It might be better to return the blank canvas from the start-designs API by intent
```

Comment:
Now that I think about it, I believe it is better now if the API returns the blank-canvas theme:

1. The API can return it based on intent
2. The API can put the blank canvas on position 3. (We don't need to sort the designs on Calypso)
3. Blank canvas theme already has `categories: []`, so we don't need to do anything special when filtering designs for SHOW_ALL_SLUG

Basically with this, Calypso does not need to do any special handling for blank canvas (other than rendering the PA component).

What do you think? I can work on it if we want.

Reply:
Sounds good! I'll create another PR to remove the logic of the blank canvas injection

Reply:
Okay, I'll let you work on it 😄